src/main/java/org/assertj/core/api/AbstractCharSequenceAssert.java

```diff
@@ -156,13 +156,13 @@ public S hasSize(int expected) {
   /**
    * Verifies that the actual {@code CharSequence} has the expected line count.
    * <p/>
-   * A line is considered to be <a name="lt">terminated</a> by any one of a line feed ('\n'), a carriage return ('\r'),
+   * A line is considered to be <a name="lt">terminated</a> by any one of a line feed ('%n'), a carriage return ('\r'),
```

Comment:
Yep, my bad… and I think this is the case for all comments right ? Including in src/main/java/org/assertj/core/error/ShouldBeInstance.java param description.